src/software/sensor_fusion/detection.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <vector>
+
```

Comment:
could we split this into `ball_detection.h`, `robot_detection.h`, `vision_detection.h`? I know some of these headers will be quite short, but IMO that's all for the better!

The reason I'm asking for it here and not for something like `TeamInfo` down below is that there are things that do not depend on everything in this header file (for example, `ball_filter.h`) but have to include everything in this header.

Reply:
fixed